python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py

```diff
@@ -236,14 +236,9 @@ def determinePsf(self, exposure, psfCandidateList, metadata=None, flagKey=None):
             gain = 1.0
             self.log.warn("Setting gain to %g" % (gain,))
```

Comment:
For consistency log.warn("Setting gain to %g", gain) ?

Reply:
Haha! Good catch, but I'm leaving it. I think the intention here is to warn if it couldn't get a detector. Whether that ever happens any more, I truly doubt, but I'm not going to change it here.